pulpcore/cli/rpm/repository.py

```diff
@@ -46,8 +46,8 @@
 translation = get_translation(__name__)
 _ = translation.gettext
 
-
-SKIP_TYPES = ["srpm"]
+# TODO: how do we limit this to just "srpm" pre-rpm-3.19?
```

Comment:
I'd give the option to the user at all times and add a check in `RpmRepositoryContext.sync`.

Reply:
Ah, ok, sounds like a plan.